ts/features/bonus/cgn/screens/merchants/CgnMerchantDetailScreen.tsx

```diff
@@ -152,10 +152,13 @@ const CgnMerchantDetailScreen: React.FunctionComponent<Props> = (
               url => (
                 <FooterWithButtons
                   type={"SingleButton"}
-                  leftButton={confirmButtonProps(
-                    () => openWebUrl(url),
-                    I18n.t("bonus.cgn.merchantDetail.cta.label")
-                  )}
+                  leftButton={{
+                    bordered: true,
+                    ...confirmButtonProps(
+                      () => openWebUrl(url),
+                      I18n.t("bonus.cgn.merchantDetail.cta.label")
+                    )
+                  }}
```

Comment:
```suggestion
                  leftButton={{
                    ...confirmButtonProps(
                      () => openWebUrl(url),
                      I18n.t("bonus.cgn.merchantDetail.cta.label")
                    ),
                    bordered: true
                  }}
```
If you want to be sure `bordered` is `true` in your props, it should be after the object spread. This is because if someone changes `confirmButtonProps` by adding `bordered: true`, it will be overwritten

![Schermata 2022-02-02 alle 18 15 45](https://user-images.githubusercontent.com/822471/152203684-90dfa7c5-42b4-42f0-86f6-8a462d3bd480.png)

Reply:
it was potentially a crack, I just didn't think about it

c3b0b91 includes the fix